### PR TITLE
add pool parameter

### PR DESCRIPTION
**Pull request by mstrzele**
closes GH-221

### CODE REVIEW

**Comment by mmoll** [comment]
After some googling, I think sqlite and mysql2 can also use pools in Rails and we should just set the default to 5 and insert that value for all DB types in database.yml.
Also, this needs a rebase. Ping @mstrzele

**Comment by mstrzele** [comment]
Rebase done. I've added a default value for connection pool.

File: manifests/init.pp [hunk 1]
```diff
@@ -78,6 +78,8 @@
 #
 # $db_sslmode::               Database 'production' ssl mode
 #
+# $db_pool::                  Database 'production' size of connection pool
+#
```
Comment (mmoll) [inline manifests/init.pp]: please add ```type:integer``` here, as seen for db_port

**Comment by mmoll** [comment]
Apart from the nitpick: tested, works for me, :+1:.
@mstrzele Please squash the commits together into one.

**Comment by mstrzele** [comment]
OK, all commits are squashed into a single commit.

**Comment by ekohl** [comment]
:+1: 

/cc @domcleal

**Comment by mmoll** [comment]
:+1:

**Comment by domcleal** [comment]
Thanks @mstrzele!

**Pull request merged**